blocks/store/reducer.js

```diff
@@ -80,10 +80,8 @@ export const fallbackBlockName = createBlockNameSetterReducer( 'SET_FALLBACK_BLO
  * @return {Object} Updated state.
  */
 export function categories( state = DEFAULT_CATEGORIES, action ) {
```

Comment:
Should we want to keep the `DEFAULT_CATEGORIES` in the client?

Reply:
That question will come up with the blocks as well. I think we need a way to generate scripts bundled as separate modules in case someone wants to use Gutenberg fully client-side.

Reply:
I think it seems fine to have a default set in the client. From purely a maintainability perspective, it would be nice if we could avoid maintaining two separate sets of the same values though (e.g. somehow inject into the client JS from the server default value, or vice-versa).

Reply:
Yes, I thought about that, too. It would be nice to have it in one place, but it is tricky because of translations. I opted for duplication to move on.  I open a follow-up issue where we can discuss how to tackle the standalone case.

Reply:
There's already some previous work done by @aduth `bin/get-server-blocks.php` that can achieve something close to what we need.